how-to/customize-workspace/client/src/settings.ts

```diff
@@ -74,9 +87,19 @@ export function validateThemes(themes: CustomThemes) : CustomThemes {
                 // don't pass an empty object as there are no theme properties
                 themeToValidate.palette = undefined;
             }
-            validatedThemes.push(themeToValidate);
+            if(themeToValidate.label.toLowerCase() === preferredColorScheme) {
+                // do not add it to the array. Hold it back to push to the front of the list of themes for default selection
+                defaultTheme = themeToValidate;
```

Comment:
Why store the default theme and add it to the list later, why not just do the unshift at this point.